### DISCUSSION TITLE
Is there a suggested way to keep web3auth session among multiple subdomains?

### DISCUSSION THREAD

**Discussion by jisoo-3pm** [Web3Auth/Web3Auth#569] [Help]
My company website is composed of three different subdomains.
And I found that web3auth session is not kept among them.
Perhaps that's because it uses localStorage.
But is there a recommended way to solve this issue?

**Comment by himanshuchawla009** [comment]
- There is a way to achieve this by syncing local storage across your sub-domains but you have to implement it on your end:-
    a. Keep the same web3auth clientId in all domains.
    b. Deploy a new subdomain for ex: auth.yourdomain.com
    c. Use auth.yourdomain.com as an iframe in your other subdomains and sync local storage data from your application subdomain to  auth.yourdomain.com using window post messages between iframe and parent. This way you can keep local storage synced between all your subdomains

  < = > : Two way channel b/w iframe and parent
 Subdomain A <=> Auth Domain (iframe) < = > Subdomain B
